src/qtiItem/core/Loader.js

```diff
@@ -204,8 +205,24 @@ var Loader = Class.extend({
                     }
                 }
 
+                const responseIdentifiers = Object.keys(this.item.responses || {})
+                    .map((responseKey) => this.item.responses[responseKey].attributes.identifier);
+
                 if (data.responseProcessing) {
-                    const customResponseProcessing = responseRules.length > 0
+                    const customResponseProcessing =
+                        (
+                            responseRules.length > 0
+                            && !(
+                                perInteractionRP
+                                && responseRules.length === 1
+                                && _.isEqual(
+                                    responseRules[0],
+                                    itemScoreHelper(
+                                        responseIdentifiers
+                                    )
+                                )
+                            )
+                        )
```

Comment:
```suggestion
                        (
                            responseRules.length > 0
                            && !(
                                responseRules.length === 1
                                && _.isEqual(
                                    responseRules[0],
                                    itemScoreHelper(
                                        responseIdentifiers
                                    )
                                )
                            )
                        )
```

Reply:
without check for `perInteractionRP`, per interaction response processing will be recognized as `templateDrive` when the feature disabled while it should be recognized as `custom`